static/js/graphs_page.js

```diff
@@ -516,6 +523,34 @@ var graphPage = {
 
         graphPage.defaultLayoutWidget.init();
     },
+    fork: function (data) {
+        //Read the meta_data object and add 'parent_id' & 'parent_email' to the data field.
+        graph_meta_data = cytoscapeGraph.getNetworkAndViewJSON(graphPage.cyGraph);
+        graph_meta_data.data['parent_id'] = data.graph_id;
+        graph_meta_data.data['parent_email'] = data.owner_email;
+        var graphData = {
+            'name':data.graph_name,
+            'is_public':0,
+            'owner_email':data.uid,
+            'graph_json':JSON.stringify(graph_meta_data, null, 4),
+            'style_json':JSON.stringify(cytoscapeGraph.getStyleJSON(graphPage.cyGraph), null, 4)
+        }
+        $.ajax({
```

Comment:
Instead of making an ajax call here. Can you add an API function at the top of the file (checkout Line 5)- just like - `apis.graphs.get` etc.

Reply:
Thanks for the review. I will make the changes you have requested and send another pull request.